Makefile

```diff
@@ -4,7 +4,7 @@ test:
 	cargo test
 
 clippy:
-	cargo clippy --all-targets --all-features -- -D warnings
+	cargo clippy --all-targets --all-features -- -W clippy::as_conversions -D warnings
```

Comment:
We should better add the lint in the `lib.rs` files (for each lib in the workspace). The idea is to add more lints in the future; if we add them to this command, it will become too large.